fe/fe-core/src/main/java/org/apache/doris/nereids/jobs/executor/Rewriter.java

```diff
@@ -204,7 +204,8 @@ public class Rewriter extends AbstractBatchJobExecutor {
                                  *  TODO: group these rules to make sure the result plan is what we expected.
                                  */
                                 new CorrelateApplyToUnCorrelateApply(),
-                                new ApplyToJoin()
+                                new ApplyToJoin(),
+                                new NormalizeAggregate()
```

Comment:
why need this normalize aggregate?

Reply:
UnCorrelatedApplyAggregateFilter rule will create new aggregate outputs, and the later rule CheckPrivileges which inherent from ColumnPruning depends on NormalizeAggregate.  There may be some better way to make it work, but correctly, use NormalizeAggregate for simplicity.

Reply:
why ColumnPruning depends on NormalizeAggregate?